src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -5,10 +5,34 @@
 	 * ----------------------------------------------------------------------------- */
 
 	.tribe-events-calendar-month__event-multiday {
-		border-radius: 6px;
-		margin-bottom: var(--spacer-0);
-		background-color: white;
-		overflow: hidden;
+		display: none;
+
+		@media (--viewport-medium) {
+			display: block;
+			border-radius: 6px;
+			background-color: white;
+			overflow: hidden;
+			position: absolute;
+			z-index: calc( var(--z-index-spinner-container) - 1 );
```

Comment:
question: do we need a positive `z-index` here, or can we get away with just `position: absolute`? are we trying to place it above something else?

Reply:
https://www.loom.com/share/e9b86afcfc6643328a6afd0fde3be2ab

Reply:
Ah that's because of the `position: relative` on the next cell's wrapper. Not having a `z-index` should not be a problem once we put in the JS, I can handle that when we merge everything with the JS.

Reply:
Awesome 🎉